scrapy/downloadermiddlewares/cookies.py

```diff
@@ -17,6 +18,7 @@ class CookiesMiddleware(object):
     def __init__(self, debug=False):
         self.jars = defaultdict(CookieJar)
         self.debug = debug
+        self.re_cookie_header = re.compile(b';\s*')
```

Comment:
```suggestion
        self.re_cookie_header = re.compile(br';\s*')
```

Otherwise, I think it will result in a warning due to `\s`.

I also wonder if it would make sense to add `\s*` before the semicolon as well.

Reply:
Changed the approach, not using `re` anymore, `bytes.split` and `bytes.strip` are more efficient in this case.